torch/lib/TH/generic/THTensorMath.c

```diff
@@ -3127,18 +3127,25 @@ accreal THTensor_(meanall)(THTensor *tensor)
   return THTensor_(sumall)(tensor)/THTensor_(nElement)(tensor);
 }
 
-accreal THTensor_(varall)(THTensor *tensor)
+accreal THTensor_(varall)(THTensor *tensor, int flag)
 {
   accreal mean = THTensor_(meanall)(tensor);
   accreal sum = 0;
   TH_TENSOR_APPLY(real, tensor, sum += (*tensor_data - mean)*(*tensor_data - mean););
-  sum /= (THTensor_(nElement)(tensor)-1);
+  if (flag)
+  {
+    sum /= THTensor_(nElement)(tensor);
+  }
+  else
+  {
+    sum /= (THTensor_(nElement)(tensor)-1);
+  }
```

Comment:
how about `sum /= THTensor_(nElement)(tensor) - (unbiased ? 1 : 0)` <- please change `flag` to `unbiased`